scala/private/phases/phase_default_info.bzl

```diff
@@ -31,11 +33,11 @@ def phase_default_info(ctx, p):
         external_providers = {
             "DefaultInfo": DefaultInfo(
                 executable = executable,
-                files = depset(transitive = files),
+                files = depset(direct = direct, transitive = files),
                 # TODO:
                 # Per Bazel documentation, we should avoid using collect_data. The core phases need to be updated
                 # before we can make the adjustment.
-                runfiles = ctx.runfiles(transitive_files = depset(transitive = runfiles), collect_data = True),
+                runfiles = ctx.runfiles(transitive_files = depset(direct = direct, transitive = runfiles), collect_data = True),
```

Comment:
Following https://groups.google.com/g/bazel-discuss/c/oEyEIr7vaF4/m/0E5QaK46AQAJ it also needs to be in runfiles.